test/test_parmed_formats.py

```diff
@@ -1581,6 +1581,18 @@ def test_download(self):
         self.assertRaises(ValueError, lambda: download_CIF('illegal'))
         self.assertRaises(IOError, lambda: download_CIF('#@#%'))
 
+    def test_cif_symmetry(self):
+        """ Tests that symmetry is parsed from mmCIF files correctly """
+        self.assertEqual(download_CIF('1aki').space_group, 'P 21 21 21')
+
+    def test_cif_space_group_written_from_structure(self):
+        parm = pmd.load_file(get_fn('SCM_A.pdb'))
+        self.assertEqual(parm.space_group, 'P 1 21 1')
+        written = get_fn('test.cif', written=True)
+        parm.write_cif(written)
+        parm2 = pmd.load_file(written)
```

Comment:
I added a test here for space_group (converted from pdb format).

However, got error
```bash
            symmetry_obj = cont.getObj('symmetry')
            if symmetry_obj is not None:
>               struct.space_group = symmetry_obj.getRow(0)[1]
E               IndexError: list index out of range
```